src/java.base/share/classes/javax/crypto/CryptoAllPermission.java

```diff
@@ -123,18 +123,18 @@ final class CryptoAllPermissionCollection extends PermissionCollection
     private boolean all_allowed;
 
     /**
-     * Create an empty CryptoAllPermissions object.
+     * Create an empty {@code CryptoAllPermission} object.
```

Comment:
typo: CryptoAllPermissions -> CryptoAllPermission?

Reply:
It's already fixed.